### DISCUSSION TITLE
当无法访问官方 / 文档中心 / 插件中心的解决方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2990] [重要通知]
## 问题描述

> 当访问 `simpread.pro` 相关网站遇到**无法访问** 或类似下图的问题

![image](https://user-images.githubusercontent.com/81074/139523202-cc572efe-3d98-4cac-810f-bc7c6cee7a64.png)

## 解决方案

> 请更新你的 DNS

```
1、114DNS

公共DNS服务器IPv4地址

首选地址114.114.114.114
备用地址114.114.115.115

2、 DNSPod Public DNS

公共DNS服务器IPv4地址

首选地址119.29.29.29
备用地址119.28.28.28

3、Baidu Public DNS

百度DNS服务器IPv4地址

180.76.76.76

4、AliDNS

阿里公共DNS服务器IPv4地址

首选地址223.5.5.5
备用地址223.6.6.6

5、oneDNS

公共DNS服务器IPv4地址

首选地址117.50.11.11
备用地址117.50.22.22
```

> 把 `github.com` `githubusercontent.com` `githubassets.com` 加入套餐

**Comment by animerqi** [comment]
![image](https://user-images.githubusercontent.com/84447025/141884098-c6d92a8d-2d88-480b-a973-6e8b8dddaac2.png)
即使更改了DNS任然没有好转，其他的几个DNS也都试过了

**Reply by Kenshin** [reply]
emmm 

一般 `114.114.114.114` 不好用的话，阿里的也肯定没问题。请问你是哪里的网络环境？

**Comment by liruiqiang** [comment]
mac 系统，更新了DNS，依然有这个问题，无法安装插件。

详细讨论 https://github.com/Kenshin/simpread/issues/4395

**Comment by idanieltam** [comment]
人在美国 用的是Verizon 加了上述所有的DNS都没有效果

**Reply by Kenshin** [reply]
嗨，看这里的解决方案 https://github.com/Kenshin/simpread/discussions/2990#discussioncomment-3612914

**Comment by Kenshin** [comment]
# 国外个别用户无法使用的解决方案

因 `cdn.jsdelivr.net` 无法访问所以把地址改为专门用于国内的 `cdn.bootcdn.net` 但近期个别国外用户无法访问 `cdn.bootcdn.net` 造成了 https://github.com/Kenshin/simpread/discussions/2990#discussion-3657766 的问题。

# 自测

请试着访问 [此链接](https://cdn.bootcdn.net/ajax/libs/font-awesome/6.1.1/css/brands.min.css) 如无法访问，则是此问题。


# 解决方案

## 文档中心

为方便外网用户，简悦官方发布了文档中心的镜像站，访问地址 http://ksria.com/simpread/docs2

## 插件中心

暂时没有镜像站方案，但可以通过直接查看 [插件说明](https://github.com/Kenshin/simpread/discussions/categories/%E6%8F%92%E4%BB%B6%E8%AF%B4%E6%98%8E) 来安装插件，随便进入一个插件说明

![image](https://user-images.githubusercontent.com/81074/189466672-08ede4ff-8590-4ccd-aafd-d07640e74f72.png)

然后通过打开链接直接安装

![image](https://user-images.githubusercontent.com/81074/189466700-25b2d6c5-01f4-4343-b9c6-59ecfca5655f.png)
